src/Nethermind/Nethermind.Db/Metrics.cs

```diff
@@ -91,5 +93,8 @@ public static class Metrics
         
         [Description("Number of Witness DB writes.")]
         public static long WitnessDbWrites { get; set; }
+
+        [Description("test.")]
+        public static ConcurrentDictionary<string, long> MetricsDict { get; set; } = new ConcurrentDictionary<string, long>();
```

Comment:
Make it IDictionary, we don't need to leak implementation, maybe call it DbStats? Do we need Description?

Reply:
Changed